README.md

```diff
@@ -1,5 +1,5 @@
 <!--
-     Copyright 2017, Data61, CSIRO (ABN 41 687 119 230).
+     Copyright 2024, seL4 Project a Series of LF Projects, LLC..
```

Comment:
You can't legally remove existing copyright holders, except when:
- You have the permission of the original copyright holders.
- Or all original content has been replaced.
- Or the copyright statement was added in error.

It is okay to add copyright holders when changing stuff, assuming the license permits changes. It's also legal to change or add a license if the new one is compatible with the original license, but you have to be very careful when doing this.

Reply:
D'oh, that was a replace all, when I erroneously changed the copyright to LF 2024. Thanks for your clarification on why not to do this. I'll revert to old copyright for all the files.